docs/userguide/requirements.txt

```diff
@@ -33,5 +33,5 @@ sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.5
 typing_extensions==4.4.0
-urllib3==1.26.12
+urllib3==2.5.0
```

Comment:
I am fine with this change as long as it does not break building documentation...

This being said, I am not sure if there is any real reason for bumping urllib3 version. There does not seem to be any security vulnerability.